lib/vagrant-openshift/action/install_origin_base_dependencies.rb

```diff
@@ -217,8 +217,8 @@ def call(env)
 fi
 if [ -n "${VG}" ]
 then
-  sudo lvcreate -n docker-data -l 90%FREE /dev/${VG}
-  sudo lvcreate -n docker-metadata -l 50%FREE /dev/${VG}
+  sudo lvcreate -n docker-data -l 70%FREE /dev/${VG}
+  sudo lvcreate -n docker-metadata -l 17%FREE /dev/${VG}
```

Comment:
I'm also going to add a `docker inspect` call to our diagnostics around
disk usage to track this moving forward.

On Tue, Jul 5, 2016 at 2:08 PM, Gabe Montero <gmontero@redhat.com> wrote:

> Given the typical 20G total, that should be 17% of 6G, or 1/6 of 6G, or
> 1G, which is the allocation it gets today.
>
> Now, running `docker inspect` on some ec2 instances, I'm only seeing about
> 4M of that 1G getting used.
>
> On Tue, Jul 5, 2016 at 1:55 PM, Ben Parees <notifications@github.com>
> wrote:
>
>> In lib/vagrant-openshift/action/install_origin_base_dependencies.rb
>> <https://github.com/openshift/vagrant-openshift/pull/466#discussion_r69608231>
>> :
>>
>> > @@ -217,8 +217,8 @@ def call(env)
>> >  fi
>> >  if [ -n "${VG}" ]
>> >  then
>> > -  sudo lvcreate -n docker-data -l 90%FREE /dev/${VG}
>> > -  sudo lvcreate -n docker-metadata -l 50%FREE /dev/${VG}
>> > +  sudo lvcreate -n docker-data -l 70%FREE /dev/${VG}
>> > +  sudo lvcreate -n docker-metadata -l 17%FREE /dev/${VG}
>>
>> we're confident 17% is enough of 30%(?) is going to be enough?
>>
>> —
>> You are receiving this because you authored the thread.
>> Reply to this email directly, view it on GitHub
>> <https://github.com/openshift/vagrant-openshift/pull/466/files/47025481713daba28b1ea2c4f6883a232563cb4e#r69608231>,
>> or mute the thread
>> <https://github.com/notifications/unsubscribe/ADbadM3jSqE9cTy440yDJ0POa46U5wFVks5qSpqQgaJpZM4JFWhN>
>> .
>>
>
>